### DISCUSSION TITLE
Could not create a module publisher account

### DISCUSSION THREAD

**Discussion by JamesKm08** [aptos-labs/aptos-developer-discussions#547] [Questions] [answered]
### Discord user ID

.jim08

### Describe your question in detail.

Is the server down? 
Creating a module publish account is not working when creating a new project(create-aptos-dapp). 
What might be the issue?

### What error, if any, are you getting?

Error: AptosApiError: Request to [Faucet]: POST https://faucet.testnet.aptoslabs.com/fund failed with status: Bad Gateway(code:502) and response body: "\n<html><head>\n<meta http-equiv=\"content-type\" content=\"text/html;charset=utf-8\">\n<title>502 Server Error</title>\n</head>\n<body text=#000000 bgcolor=#ffffff>\n<h1>Error: Server Error</h1>\n<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>\n<h2></h2>

### What have you tried or looked at?  Or how can we reproduce the error?

Just using the create-aptos-dapp wizard when starting out and this is the error. Everything is created other than the publisher account which is saying I need to add it manually in my .env file

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

Aptos CLI

### Describe your environment or tooling in detail

Linux OS, aptos version 4.5.0

**Comment by 0xmaayan** [comment, marked as answer]
Hi @JamesKm08 testnet faucet is currently down due to an issue we are currently investigating. As a workaround please share the account address you use as the publisher account and we can send test APT to that account

**Comment by JamesKm08** [comment]
Here is my address : 0xc1b6c3ce53147deed192b3e6c179aa54560761edf28c81d78c0c614b235c7941 . Thanks for the help

**Reply by 0xmaayan** [reply]
Hi, thanks! Testnet faucet is online at https://aptos.dev/en/network/faucet
